MatrixSDK/VoIP/CallStack/Jingle/MXJingleCallAudioSessionConfigurator.m

```diff
@@ -0,0 +1,181 @@
+/*
+ Copyright 2017 Vector Creations Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MXJingleCallAudioSessionConfigurator.h"
+
+#if defined MX_CALL_STACK_JINGLE && TARGET_OS_IPHONE
+
+@import AVFoundation;
+
+#import <objc/message.h>
+#import <objc/runtime.h>
+#import <sys/utsname.h>
+
+// Preferred hardware sample rate (unit is in Hertz). The client sample rate
```

Comment:
Could you please add a link to https://chromium.googlesource.com/external/webrtc/+/master/webrtc/sdk/objc/Framework/Classes/Audio/RTCAudioSessionConfiguration.m and explain why we temporary copy the code (basically what you wrote in the PR submission)?

The code is under BSP licence, we can copy part of it but we must give them credits.